### PR TITLE
private(set) var Component.state 

### PR DESCRIPTION
**Pull request by adly-holler**
Right?

### CODE REVIEW

File: FewCore/Component.swift [hunk 1]
```diff
@@ -25,7 +25,7 @@ import AppKit
 /// by calling the `render` function passed in to its init. But subclasses can
 /// optimize this by implementing `componentShouldRender`.
 public class Component<S>: Element {
-	private var state: S
+	private(set) var state: S
```
Comment (robrix) [inline FewCore/Component.swift]: I always forget you can do this.

**Comment by joshaber** [comment]
Whaaaaaaat. So this makes `state` publicly readable by only privately settable?

**Comment by adly-holler** [comment]
Yezzur! We live in an incredible age

File: FewCore/Component.swift (hunk 1 above)
Comment (joshaber) [inline FewCore/Component.swift]: Damn, nice. Mind doing `public private(set)` to match Element et al?

**Comment by adly-holler** [comment]
Oh yeah it might be good if users could read component states, I guess.

**Comment by joshaber** [comment]
:boom:

**Pull request merged**